conformance/conformance.proto

```diff
@@ -57,11 +57,13 @@ option java_package = "com.google.protobuf.conformance";
 //   2. parse the protobuf or JSON payload in "payload" (which may fail)
 //   3. if the parse succeeded, serialize the message in the requested format.
 message ConformanceRequest {
+  string test_name = 1;
```

Comment:
Is the testee supposed to make use of this name somehow? It seems to me this name is only useful in the test-runner and there is no need to include it in the request message.